conjure-undertow-annotations/src/main/java/com/palantir/conjure/java/undertow/annotations/ParamDecoders.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.undertow.annotations;
+
+import com.google.common.collect.Iterables;
+import com.palantir.logsafe.exceptions.SafeIllegalArgumentException;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public final class ParamDecoders {
+
+    public static ParamDecoder<String> stringDecoder() {
+        return DelegateDecoder.of(Function.identity());
```

Comment:
The actual conversion is quite simple but maybe we want to delegate to [PlainSerde](https://github.com/palantir/conjure-java/blob/6866283be95e05d123af57033e3867b18bda7ed5/conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjurePlainSerDe.java) for the types that are supported? Doesn't make the code shorted but that way we would have consistent error handling.

Reply:
I like it, we'd exactly match conjure-undertow where possible

Reply:
Updated the default decoders to delegate to the respective method of `PlainSerDe`. Will add the missing methods in a flup commit so we support the same types as `PlainSerDe` out-of-the-box.